new-log-viewer/src/typings/decoders.ts

```diff
@@ -42,33 +49,49 @@ interface Decoder {
     getEstimatedNumEvents(): number;
 
     /**
-     * When applicable, deserializes log events in the range `[beginIdx, endIdx)`.
+     * @return Indices of the filtered events.
```

Comment:
```suggestion
     * @return The filtered log events map.
```

I think the type name is descriptive enough.